app/models/media.rb

```diff
@@ -326,4 +326,20 @@ def archive_if_conditions_are_met(options, id, cache)
         self.archive(options.delete(:archivers))
     end
   end
+
+  def parser_requests_metrics
+    url = RequestHelper.normalize_url(self.url)
```

Comment:
Question: Isn't the URL already normalized at this point? In any case, it should be harmless if the normalization function is idempotent.

Reply:
I think it should be, but some tests were failing, seeming to indicate it wasn't (which I found weird). Maybe it's an issue on the tests side. Going to double check this one.

Reply:
Updating with the answer from our discussion:

> We do normalize the URL for parsing BUT we return the original URL. Me and Jay think that makes sense: the original URL is readable, and it might look weird for the user to send one URL in and get another. But we wanted to confirm with you if our assumptions make sense. So we do need to normalize `data[:url]` so we can get the host for our metrics.

Also, in the comment I refer to `data[:url]` but in the code I ended up using `self.url`, since we didn't have a preference on which one to use. I thought if it doesn't really make a difference we might as well go with the original one we got.

> Is there a reason we should prefer one over the other? Conceptually what makes more sense for us?
> 
> No, I think they are the same thing. But one is “internal” (thus a model attribute) and the other is just the exposed value.